### PR TITLE
Image overlay

### LINKED ISSUE #7: Columns block: Image overlay
Allow images in a columns block to overlay each other and expand outside of the columns container.

image

### PR DESCRIPTION
**Pull request by allentsliu**
## Issue

Fix #7 

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1
- After: https://imageoverlay--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1 

## 📝 Description:
add image overlay for the image-overlay column block

Screen Shot 2023-05-30 at 2 48 30 PM

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -28,4 +28,29 @@
...
+  flex-grow: 2;
+}
+
+.image-overlay  > div > div:first-child {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}
+
+.image-overlay img{
```
Comment (tmathern) [inline blocks/columns/columns.css:43]:
```suggestion
.image-overlay img {
```

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -28,4 +28,29 @@
...
+  position: relative;
+  max-width: 60%;
+  max-height: auto;
+}
+
+.image-overlay > div > div > p:last-child img{
+  height: 95%;
+}
+
+.image-overlay > div > div > p:first-child img{
```
Comment (tmathern) [inline blocks/columns/columns.css:53]:
```suggestion
.image-overlay > div > div > p:first-child img {
```

**Review by tmathern — APPROVED** [review]
Approving for the images.
Some notes:
- In their webpage, the bottles seem to have a slightly different placement, as the top layer-one is in front and slightly larger than th background one. I think the intend is to focus on the front bottle. We could probably bake it in in the CSS, if we want to copy that effect 1:1:
Screenshot 2023-05-30 at 11 02 45 AM

File: blocks/columns/columns.css [hunk 3]
```diff
@@ -28,4 +28,29 @@
 
 .columns > div > .columns-img-col img {
   display: block;
+}
+
+.image-overlay > div > div:last-child {
```
Comment (arumsey) [inline blocks/columns/columns.css:33]: Add `.columns.image-overlay` so this style only gets applied to the columns block.

File: blocks/columns/columns.css [hunk 4]
```diff
@@ -28,4 +28,29 @@
...
+  flex-grow: 2;
+}
+
+.image-overlay  > div > div:first-child {
```
Comment (arumsey) [inline blocks/columns/columns.css:37]: Remove extra space before `>`

**Comment by arumsey** [comment]
Will this PR also address the image overflowing outside of the columns block? This may need some absolute positioning to achieve that effect.

image

**Comment by allentsliu** [comment]
@arumsey I am able to make the image overflow outside the column block by setting the top property to a negative number (i.e `top:-100px`) 
Currently, the issue is that the text size appear smaller on our page compare to the live customer page even though they seem to have the same font-size. So, it's not easy to put the image at the right location. I am trying to figure out the text size issue.

**Comment by tmathern** [comment]
@arumsey @allentsliu You may need to set a fixed (min)height to be able to have the image overflow nicely.

**Comment by allentsliu** [comment]
@arumsey @tmathern  thank you for the review. for the text size issue,  I can't seem to find the root cause of the issue. So, I increase the size in the style.css file. Is it a proper fix? If the change looks good, are you fine with merging this PR? Thanks!

**Comment by arumsey** [comment]
> @arumsey @tmathern thank you for the review. for the text size issue, I can't seem to find the root cause of the issue. So, I increase the size in the style.css file. Is it a proper fix? If the change looks good, are you fine with merging this PR? Thanks!

We can address text size and overall screen size issues in a future PR once we determine what our responsive strategy should be.

**Review by arumsey — APPROVED** [review]

**Pull request merged**